src/main/java/org/cs1331/gitsubmitter/Utils.java

```diff
@@ -227,6 +229,32 @@ private static void decrypt(File file, File out, File aesIn,
         cos.close();
     }
 
+    /**
+     * Attempts to retrieve a unique identifier for this computer.
+     * The MAC address of the currently active network interface is used
+     * as this identifier. If the MAC address is unable to be retrieved,
+     * "000000000000" is returned.
+     * 
+     * @return A unique identifier of this computer
+     */
+    public static String getIdentifier() {
+        byte[] id;
```

Comment:
That's handled. NetworkInterface.getHardwareAddress() will either return a byte array or null, and if an exception occurs, id is explicitly set to null.